src/lib/dune

```diff
@@ -7,7 +7,7 @@
 (library
   (name AltErgoLib)
   (public_name alt-ergo-lib)
-  (ocamlopt_flags (:standard -open Prelude))
+  (flags (:standard -open Alt_ergo_prelude__Prelude))
```

Comment:
I think it would be better to keep `(wrapped false)`, rename `prelude.ml` to `alt_ergo_prelude.ml`, and use `-open Alt_ergo_prelude` here; this would avoid depending on the mangling performed by `(wrapped true)`.

Reply:
I have a better option: I kept `(wrapped true)` but I rename `prelude.ml` to `alt_ergo_prelude.ml`. In this case, I don't need to mangle the module in `(flags ...)`.

Reply:
Does that work? I thought `(wrapped false)` was introduced because that was not working?

Reply:
OK, then that's the best solution indeed.